admin/tabs/cluster/ClusterTabModel.ts

```diff
@@ -80,7 +91,7 @@ export class ClusterTabModel extends HoistModel {
             },
             {
                 track: () => XH.environmentService.serverInstance,
-                run: () => this.gridModel.agApi.refreshCells({force: true})
+                run: this.refreshAsync
```

Comment:
We could consider nixing this now given the overall autorefresh.  You will be getting a refresh within 5s anyway, usually within 3s. Feels like "close enough" - maybe just on the line.

Reply:
True - I guess just depends on whether we care about the grid and version bar getting out of sync for a couple seconds.

Reply:
happy to leave it as-is - since it's async it's never going to be instant anyway, but doesn't seem like there's a real cost here, no need to over-think in either direction

Reply:
great points, especially with the inline error and not refreshing when hidden